include/nbl/asset/filters/CCopyImageFilter.h

```diff
@@ -64,8 +65,9 @@ class NBL_API CCopyImageFilter : public CImageFilter<CCopyImageFilter>, public C
 				const auto blockDims = asset::getBlockDimensions(commonExecuteData.inFormat);
 				auto copy = [&commonExecuteData,&blockDims](uint32_t readBlockArrayOffset, core::vectorSIMDu32 readBlockPos) -> void
 				{
-					auto localOutPos = readBlockPos*blockDims+commonExecuteData.offsetDifference;
-					memcpy(commonExecuteData.outData+commonExecuteData.oit->getByteOffset(localOutPos,commonExecuteData.outByteStrides),commonExecuteData.inData+readBlockArrayOffset,commonExecuteData.outBlockByteSize);
+					const auto localOutPos = readBlockPos+commonExecuteData.offsetDifference;
+					const auto writeOffset = commonExecuteData.oit->getByteOffset(localOutPos,commonExecuteData.outByteStrides);
+					memcpy(commonExecuteData.outData+writeOffset,commonExecuteData.inData+readBlockArrayOffset,commonExecuteData.outBlockByteSize);
```

Comment:
did something change, or is this only formatting?

Reply:
I fixed the bug, I replaced "readBlockPos*blockDims" with "readBlockPos"